File: sub-web-model/src/main/java/org/github/base/entity/ArticleTagEntity.java
package org.github.base.entity;

import org.github.base.Model;
import com.baomidou.mybatisplus.annotation.*;
import java.io.Serializable;
import javax.annotation.Nullable;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-07
 */
@SuppressWarnings("all")
@Data
@EqualsAndHashCode(callSuper = true)
@TableName("article_tag")
@ApiModel(value="ArticleTagEntity对象", description="")
public class ArticleTagEntity extends Model<ArticleTagEntity> {

  /** UID */
  private static final long serialVersionUID = 1L;

  @TableId(value = "id", type = IdType.AUTO)
  private @Nullable Long id;

  /** 0：未启用	            1：已启用 */
  @ApiModelProperty(value = "0：未启用	            1：已启用")
  @TableField("tag_id")
  private @Nullable Integer tagId;

  @TableField("article_id")
  private @Nullable Long articleId;

  @Override
  protected Serializable pkVal() {
    return this.id;
  }

}

File: sub-mysql-blog-entity/src/main/java/org/github/mysql/blog/base/entity/ArticleTagEntity.kt
package org.github.mysql.blog.base.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import org.github.base.Entity
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * 
 *
 * @author JYD_XL
 */
@TableName("tb_article_tag")
@ApiModel(value = "ArticleTagEntity对象", description = "")
open class ArticleTagEntity : Entity() {

    @TableId(value = "id", type = IdType.AUTO)
    open var id: Int? = null

    /** 文章id */
    @ApiModelProperty(value = "文章id")
    @TableField("article_id")
    open var articleId: Int? = null

    /** 标签id */
    @ApiModelProperty(value = "标签id")
    @TableField("tag_id")
    open var tagId: Int? = null

    companion object {

        private const val serialVersionUID = 1L

        const val ID : String = "id"

        const val ARTICLE_ID : String = "article_id"

        const val TAG_ID : String = "tag_id"

    }

    override fun pkVal(): java.io.Serializable? {
        return id
    }

    override fun toString(): String {
        return "ArticleTagEntity{" +
        "id=" + id +
        ", articleId=" + articleId +
        ", tagId=" + tagId +
        "}"
    }

}
